org-formation/650-identity-providers/_tasks.yaml

```diff
@@ -0,0 +1,20 @@
+Parameters:
+  <<: !Include '../_parameters.yaml'
+
+GithubOidcSageBionetworksIt:
+  Type: update-stacks
+  Template: github-oidc-provider.njk
+  StackName: github-oidc-sage-bionetworks-it
+  Parameters:
+    ManagedPolicyArns:
+      - "arn:aws:iam::aws:policy/AdministratorAccess"
```

Comment:
Are we giving Github administrator access to our organization?  That seems too broad.

Reply:
GH already has admin access.  It needs that level of access to deploy/update resources defined in the templates in this repo.